views/annotation/annotationoptions.html

```diff
@@ -32,16 +45,16 @@
           </label>
         </div>
       </div>
-      <div class="col">
+      <div class="col ml-5">
```

Comment:
Is this intended to indent these color options under Show Background to convey their relationship? Personally I think it looks better with everything inline, and I don't believe we do this in any other forms. I would opt for disabling the color pickers when Show Background is unchecked.

Reply:
The legend settings used to have indentation like that ... I'm guessing that was lost with the bs4 upgrade though (as well as other examples)

Reply:
@gregroyal mentioned that we avoid indenting levels in the UI because it's easier for a user to scan if everything is lined up. I think we should do the same here.

Reply:
Extra padding has been removed.